auction-app-backend/src/main/java/com/atlantbh/auctionappbackend/HealthCheckController.java

```diff
@@ -0,0 +1,15 @@
+package com.atlantbh.auctionappbackend;
```

Comment:
We could put this class into `controller` package. Also, you can create packages for `service`, `repository` and `domain` classes, since we'll need them in the future.

Reply:
Sure, we'll structure the project by layers.